tests/functional/requirements.txt

```diff
@@ -0,0 +1,5 @@
+PyJwt
```

Comment:
Delete file please, use `poetry add --dev <dependency>` to add things to the pyproject.toml